releaseNotes/master.md

```diff
@@ -4,28 +4,30 @@
 ${REL_VER_DATE}
 
 ## Features
-  - **simple title**: brief description. [link to docs](#).
-      - itemized
-      - list
-      - for details
-      - if necessary
+  - **SPOG subscription level object limit**: Subscription administrators can now set a limit to the number of objects that can be displayed in the SPOG view for that subscription, this can be used to prevent users from accidentally loading the full SPOG which will be helpful for subscriptions with a very large number of resources/jobs in them which will possibly create browser performance issues.
+  this setting can be found in the subscription settings page with the title `SPOG Object Limit`.
+  - **Ability to create subscription owned integrations**: Subscription users can now create shared subscription integrations, which allows you to restrict the usage of integration to members, collaborators and admin based on your requirement. Customers using BYON should upgrade their nodes and customers using caching with on-demand nodes should stop all their nodes.  ([support#4128](https://github.com/Shippable/support/issues/4128))
```

Comment:
This feature requires v6.7.4 of the build runner. BYON nodes will need to be reinitialized if they are running older versions. Customers running on-demand nodes with caching will need to stop their cached nodes to get the latest version of the build runner.